solr/solr-ref-guide/src/other-parsers.adoc

```diff
@@ -1037,11 +1037,11 @@ An optional parameter used to determine which of several query implementations s
 +
 `termsFilter` the default `method`.  Uses a `BooleanQuery` or a `TermInSetQuery` depending on the number of terms.  Scales well with index size, but only moderately with the number of query terms.
 +
-`docValuesTermsFilter` can only be used on fields with docValues data.  Chooses between the `docValuesTermsFilterTopLevel` and `docValuesTermsFilterPerSegment` methods using the number of query terms as a rough heuristic.  Users should typically use this method instead of using `docValuesTermsFilterTopLevel` or `docValuesTermsFilterPerSegment` directly, unless they've done performance testing to validate one of the methods on queries of all sizes.  Depending on the implementation picked, this method may rely on expensive data structures which are lazily populated after each commit.  If you commit frequently and your use-case can tolerate a static warming query, consider adding one to `solrconfig.xml` so that this work is done as a part of the commit itself and not attached directly to user requests.
+`docValuesTermsFilter` can only be used on fields with docValues data.  The `cache` parameter is false by default.  Chooses between the `docValuesTermsFilterTopLevel` and `docValuesTermsFilterPerSegment` methods using the number of query terms as a rough heuristic.  Users should typically use this method instead of using `docValuesTermsFilterTopLevel` or `docValuesTermsFilterPerSegment` directly, unless they've done performance testing to validate one of the methods on queries of all sizes.  Depending on the implementation picked, this method may rely on expensive data structures which are lazily populated after each commit.  If you commit frequently and your use-case can tolerate a static warming query, consider adding one to `solrconfig.xml` so that this work is done as a part of the commit itself and not attached directly to user requests.
```

Comment:
Just a side-comment about our ref docs:  In the ref docs I write or update, I convert them to one sentence per line.  This makes the diffs easy to read!  The pain of no newlines is very apparent here.  Change or not as you wish.  CC @ctargett

Reply:
I thought about this too when I saw the diff.  I remember it coming up as a discussion point when the ref-guide was initially being hashed out too.

I won't change my docs here, as it's probably worth keeping the file as a whole consistent in the strategy it uses.  But maybe it's worth revisiting at some point.

Reply:
When I think of it, I've been changing to one sentence per line when I'm editing also (but I often forget). I don't know that it really changes that much to have a whole page consistent - we should probably encourage one sentence per line, but if it's a big file and you can only do a little, I think that's fine and maybe someone else will get inspired later to do the rest.